libraries/rush-lib/src/logic/installManager/InstallHelpers.ts

```diff
@@ -42,6 +42,35 @@ export class InstallHelpers {
     JsonFile.save(commonPackageJson, commonPackageJsonFilename, { onlyIfChanged: true });
   }
 
+  public static generateCommonSplitPackageJson(
+    rushConfiguration: RushConfiguration,
+    dependencies: Map<string, string> = new Map<string, string>()
+  ): void {
+    const commonPackageJson: IPackageJson = {
+      dependencies: {},
+      description: 'Temporary file generated by the Rush tool',
+      name: 'rush-common-split',
+      private: true,
+      version: '0.0.0'
+    };
+
+    // Add any preferred versions to the top of the commonPackageJson
```

Comment:
We should not add preferred versions to the split workspace, since the goal is to preserve the old module resolutions